src/isql/isql.h

```diff
@@ -311,4 +311,26 @@ struct IsqlVar
 	TypeMix value;
 };
 
+class IsqlWireStats
+{
+public:
+	IsqlWireStats(Firebird::IAttachment* att);
+
+	bool print(bool initial);
+	bool get(bool initial);
+
+private:
+
+	Firebird::IAttachment* m_att;
+	FB_UINT64 m_snd_packets;
```

Comment:
Isn't it more simple/modern to initialize the fields here to zero?

Reply:
Done